syft/sandbox.py

```diff
@@ -56,6 +58,24 @@ def load_sklearn(func, *tags):
 
             return data, target
 
+        def load_tf(func, *tags):
+            num_of_records = 10000
+            """Int: num_of_records is a configurable limit for the cifar10
+            and fashion_mnist datasets.
+            since it is a huge dataset and it requires a lot of memory resources"""
+
+            ((train_images, train_labels), (test_images, test_labels)) = func()
+            data = np.concatenate([train_images, test_images])
+            target = np.concatenate([train_labels, test_labels])
+
+            data = data[0:num_of_records]
+            target = target[0:num_of_records]
+
+            data = torch.IntTensor(data).tag(*(list(tags) + ["#data"])).describe(tags[0][1:])
+            target = torch.IntTensor(target).tag(*(list(tags) + ["#target"])).describe(tags[0][1:])
```

Comment:
Are you sure this should be Integers and not simple `torch.Tensor` ?

Reply:
Yes, since the datasets of cifar10 and fashion_mnist are integers (where the default for torch.Tensor is float)